Dockerfile

```diff
@@ -1,16 +1,15 @@
-ARG TAG="v1.0.0"
+ARG TAG="14bd335c17c1b4c6cb7d37c2972c05cc62cadeeb"
```

Comment:
Can you add the comment about it being a commit to this line as well?

Reply:
good point!